packages/account/src/Constants/trading-assessment-questions.tsx

```diff
@@ -0,0 +1,252 @@
+import { localize } from '@deriv/translations';
+
+export const getTradingAssessmentQuestions = () => [
+    {
+        question_text: localize(
```

Comment:
`question_text` can be converted to `<Localize/>`

Reply:
Cannot be done because shared components having different types